core/variant_op.cpp

```diff
@@ -699,7 +702,13 @@ void Variant::evaluate(const Operator& p_op, const Variant& p_a, const Variant&
 				DEFAULT_OP_FAIL(MATRIX32);
 				DEFAULT_OP_LOCALMEM_NUM(/,VECTOR3,Vector3);
 				DEFAULT_OP_FAIL(PLANE);
-				DEFAULT_OP_FAIL(QUAT);
+				case QUAT: {
```

Comment:
`DEFAULT_OP_LOCALMEM_NUM` not only provides the operation on numbers but also on the type itself. 
However, Quat / Quat is not a valid op.